objects/obj_ncombat/Alarm_0.gml

```diff
@@ -576,6 +576,258 @@ if (enemy = 2) {
 	if (threat>1){u.dudes[3]="Leman Russ Battle Tank";u.dudes_num[3]=min(1000,floor(guard_effective*0.1));enemies[3]=u.dudes[3];}*/
 }
 
+// Need to add Admech (faction 3) and Inquisition (faction 4) force rosters
+
+// ** Sisters Force **
+if (enemy = 5) {
+	// Sister Squad
+	if (threat = 1) {
+		u = instance_nearest(xxx, 240, obj_enunit);
+		enemy_dudes = "4"; // This does raise a question - should strength 1 just send in the chaff, supporting the Sisters? Or go with the actual sister squad?
+
+		u.dudes[1] = "Celestian";
+		u.dudes_num[1] = 1;
+		enemies[1] = u.dudes[1];
+		// Spawn Leader
+		if (leader = 1) {
+			u.dudes[1] = "Leader";
+			u.dudes_num[1] = 1;
+			enemies[1] = 1;
+			enemies_num[1] = 1;
+		}
+		u.dudes[2] = "Battle Sister";
+		u.dudes_num[2] = 3;
+		enemies[2] = u.dudes[2];
+	}
+	// Sister Demi-Platoon
+	if (threat = 2) {
+		u = instance_nearest(xxx + 10, 240, obj_enunit);
+		enemy_dudes = "20"; // Full formation: 1 overstrength SoB squad, with 2 specialized sisters
+
+		u.dudes[1] = "Celestian";
+		u.dudes_num[1] = 1;
+		// Spawn Leader
+		if (leader = 1) {
+			u.dudes[1] = "Leader";
+			u.dudes_num[1] = 1;
+			enemies[1] = 1;
+			enemies_num[1] = 1;
+		}
+		u.dudes[2] = "Battle Sister";
+		u.dudes_num[2] = 17;
+		u.dudes[3] = "Retributor"; // Should be made random whether it is 1 retributor or 1 dominion, maybe
+		u.dudes_num[3] = 1;
+		u.dudes[4] = "Dominion";
+		u.dudes_num[4] = 1;
```

Comment:
As a side note: this kind of asks for randomization stuff or more complex conditions:
It can instead be the surviving Retributor or Dominion sisters, repentias as well.
Also perhaps it could have a Rhino or 2, indicating a small mechanized detachment, instead or regular.
Though, the complexity is probably a bit out of scope for this, so I'll probably experiment with randomization instead.